spark-jobs/src/main/scala/za/co/absa/enceladus/HelpPrinter.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus
+
+import za.co.absa.enceladus.conformance.config.ConformanceConfig
+import za.co.absa.enceladus.standardization.config.StandardizationConfig
+import za.co.absa.enceladus.standardization_conformance.config.StandardizationConformanceConfig
+
+object HelpPrinter {
```

Comment:
I think it is better then doing a switch statement in bash. And I think this would be less straining on the system/edge node than running main spark jobs that are immediately killed.